components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.ts

```diff
@@ -40,16 +40,17 @@ export class ProjectsDropdownComponent implements OnInit, OnDestroy, OnChanges {
     this.store.select(assignableProjects)
       .pipe(takeUntil(this.isDestroyed))
       .subscribe((assignable: ProjectsFilterOption[]) => {
-        this.projects =
-          assignable.map(p => {
-            return <ResourceChecked>{
+        this.projects = [{
+          itemList: assignable.map(p => {
```

Comment:
The underlying data structure was refined to contain sections needed for policies. Projects, though, do not need multiple sections, so all data goes in one.